File: Week4/Thursday/stockpicker/js/stockPicker.js
const picker = prices => {
  let days = [0,0]
  let profit = 0
  for (let i = prices.length - 1; i >= 0 ; i--) {
      for (let j = 0; j < i; j++) {
        if ((prices[i] - prices[j]) > profit) {
          profit = prices[i] - prices[j]
          days[0] = j
          days[1] = i
        }
      }
    }
  return days

}


module.exports = {
    picker
}

File: Week4/Thursday/stockpicker/python/stock_picker.py
def picker(prices):

    days = [0,0]
    profit = 0
    for i in range(len(prices)-1, -1, -1):
        for j in range(0, i):
            if (prices[i] - prices[j]) > profit:
                profit = prices[i] - prices[j]
                days[0] = j
                days[1] = i
    return days
    
